README.md

```diff
@@ -3,18 +3,19 @@
 ## Table of contents
 
 - 1.0) [About](#10-about)
-    - 1.1) [Downloading OpenTTD](#11-downloading-openttd)
-    - 1.2) [OpenTTD gameplay manual](#12-openttd-gameplay-manual)
-    - 1.3) [Supported platforms](#13-supported-platforms)
-    - 1.4) [Installing and running OpenTTD](#14-installing-and-running-openttd)
-    - 1.5) [Add-on content / mods](#15-add-on-content--mods)
-    - 1.6) [OpenTTD directories](#16-openttd-directories)
-    - 1.7) [Compiling OpenTTD](#17-compiling-openttd)
+  - 1.1) [Downloading OpenTTD](#11-downloading-openttd)
+  - 1.2) [OpenTTD gameplay manual](#12-openttd-gameplay-manual)
+  - 1.3) [Supported platforms](#13-supported-platforms)
+  - 1.4) [Installing and running OpenTTD](#14-installing-and-running-openttd)
+  - 1.5) [Add-on content / mods](#15-add-on-content--mods)
+  - 1.6) [OpenTTD directories](#16-openttd-directories)
+  - 1.7) [Compiling OpenTTD](#17-compiling-openttd)
+  - 1.8) [Build docker image](#18-build-docker-image)
 - 2.0) [Contact and community](#20-contact-and-community)
-    - 2.1) [Multiplayer games](#21-multiplayer-games)
-    - 2.2) [Contributing to OpenTTD](#22-contributing-to-openttd)
-    - 2.3) [Reporting bugs](#23-reporting-bugs)
-    - 2.4) [Translating](#24-translating)
+  - 2.1) [Multiplayer games](#21-multiplayer-games)
+  - 2.2) [Contributing to OpenTTD](#22-contributing-to-openttd)
+  - 2.3) [Reporting bugs](#23-reporting-bugs)
+  - 2.4) [Translating](#24-translating)
 - 3.0) [Licensing](#30-licensing)
 - 4.0) [Credits](#40-credits)
```

Comment:
Why does the readme need to be completely reformatted?
If there's a good reason for it, it should ideally be a separate commit/PR.